model/mrd_protocol.yml

```diff
@@ -1,17 +1,30 @@
-WaveformUint32: Waveform<uint32>
-ImageUint16: Image<uint16>
-ImageInt16: Image<int16>
-ImageUint: Image<uint>
-ImageInt: Image<int>
-ImageFloat: Image<float>
-ImageDouble: Image<double>
-ImageComplexFloat: Image<complexfloat>
-ImageComplexDouble: Image<complexdouble>
+# Union of all primary types that can be streamed in the MRD Protocol
+StreamItem:
```

Comment:
I think we can/should later on consider having protocols dedicated to the intermediate formats. Working with those protocols in code may be easier. On the flip side, having to explicitly switch protocols is perhaps a hassle.